AnnoyancesFilter/sections/popups.txt

```diff
@@ -112,6 +112,9 @@ bankbazaar.com#$#.oneTapLoginModal[style] { display: none !important; }
 bankbazaar.com#$#.modal-backdrop { display: none !important; }
 bankbazaar.com#$##js-sign-in-box[style="display: block;"] ~ .modal-backdrop { display: block !important; }
 magadanmedia.ru###social_modalblock
+inc42.com#$#iframe[id^="offer-"] { display: none !important }
```

Comment:
Please re-add `;` after `!important`.
Other problem is that, when popup is displayed, content is not clickable due to `tp-modal` element.

Reply:
Could you please try 
`inc42.com#$#.tp-modal { pointer-events: none !important; }
`
on your side?

Reply:
It works, but if I'm not wrong, it causes that login popup is not clickable, I mean that for example you can't close it.

Reply:
yeah, just fixed it a minute ago with `pointer-events: all` where needed